src/main/java/org/ilgcc/app/inputs/Providerresponse.java

```diff
@@ -157,6 +157,11 @@ public class Providerresponse extends FlowInputs {
     //registration-home-provider-tax-id
     @NotBlank(message = "{errors.select-one-option}")
     private String homeProviderTaxIDselection;
+    
+    //registration-home-provider-itin
+    @NotBlank(message = "{registration-home-provider-itin.error-blank}")
+    @Pattern(regexp = "\\d{9}", message = "{registration-home-provider-itin.error-invalid}")
```

Comment:
Does this show a double error when you have a blank itin?

if so, you might need:

```suggestion
    @Pattern(regexp = "^$|\\d{9}", message = "{registration-home-provider-itin.error-invalid}")
```

Reply:
Confirming that it does double display so you do need to add the suggestion I made:

<img width="413" alt="Screenshot 2025-04-18 at 11 00 48 AM" src="https://github.com/user-attachments/assets/1a518574-3ba4-4877-9843-74a5534d79f5" />

Reply:
Good call, @analoo ! Yeah, let's just use 1 annotation with a better regex.